api/src/main/java/au/org/aodn/nrmn/restapi/controller/SiteController.java

```diff
@@ -51,6 +52,22 @@
         return siteRepository.findAll();
     }
 
+    @GetMapping(path = "/siteCodes")
+    public ResponseEntity<List<String>> getAllSiteAreas() {
+        return ResponseEntity.ok(siteRepository.findAllSiteCodes().stream().collect(Collectors.toList()));
```

Comment:
Why not just return a list in the repository methods and not bother converting to one here.